src/Ladder.java

```diff
@@ -0,0 +1,67 @@
+import java.util.Random;
+
+public class Ladder {
+    private int numberOfPeople;
+    private int heightOfLadder;
+    private char[][] charLadder;
+//    private StringBuilder sb;
+
+    Ladder(int n, int m) {
+        this.numberOfPeople = n;
+        this.heightOfLadder = m;
+        this.init();
+    }
+
+    private void init() {
+        this.charLadder = makeLadder();
+    }
+
+    private char[][] makeLadder() {
```

Comment:
```suggestion
    private char[][] make() {
```
메소드 이름과 클래스 이름에서 단어가 중복되는 것은 피하는 것이 좋습니다.